app/retail/templates/shared/activity.html

```diff
@@ -71,16 +103,21 @@
             <span>{{ row.value_in_token_disp }} {{row.token_name}}</span>
             {% elif row.metadata.value_in_eth %}
             <span>{{ row.metadata.value_in_eth }} {{row.token_name}}</span>
+            {% elif row.grant %}
+            {{ row.metadata.value_in_token | floatformat:2 }} / {{ row.metadata.amount_goal | floatformat:2 }}
+            {{ row.metadata.token_name }}
```

Comment:
indent the above 2 lines by 1 ?

Reply:
Natürlich!

Reply:
I just discovered that I was using HTML mode in emacs to work on these templates. That's why I got these indentation flaws.